modules/contact/test/tests/mortar_aux_kernels/tests

```diff
@@ -179,4 +184,48 @@
                   'system shall generate many segments from oblique projections if a very small '
                   'minimum projection angle is chosen. '
   []
+  [pressure-aux-frictionless]
+    type = CSVDiff
+    input = 'pressure-aux-frictionless.i'
+    csvdiff = 'pressure-aux-frictionless_out_normal_lm_0011.csv'
+    abs_zero = 1.0e-4
+    min_ad_size = '50'
+    ad_indexing_type = 'global'
+    mesh_mode = 'REPLICATED'
+    requirement = 'The MortarPressureComponentAux should produce correct normal contact pressure '
```

Comment:
Let's keep implementation details (like class names) out of requirements. Similarly for other requirements in this spec

Reply:
Done